app/models/github_analysis.go

```diff
@@ -6,7 +6,7 @@ import (
 
 //go:generate goqueryset -in github_analysis.go
 
-// gen:qs
+//gen:qs
```

Comment:
comment on exported type GithubAnalysis should be of the form "GithubAnalysis ..." (with optional leading article)

Reply:
will be fixed in the newest `golangci-lint` release